packages/belos/tpetra/src/solvers/Belos_Tpetra_Gmres.hpp

```diff
@@ -487,7 +486,9 @@ class Gmres : public Krylov<SC, MV, OP> {
     mag_type b_norm;  // initial residual norm
     mag_type b0_norm; // initial residual norm, not left preconditioned
     mag_type r_norm;
+    mag_type r_norm_imp;
     vec_type R (B.getMap ());
+    vec_type Y (B.getMap ());
```

Comment:
In the future, we should talk about reducing overhead by combining allocations.  This is fine for now, though.